hubblestack/extmods/modules/fdg.py

```diff
@@ -147,8 +146,13 @@ def run(fdg_file, starting_chained=None):
         Allows you to pass in a starting argument, which will be treated as
         the ``chained`` argument for the ``main`` block. Optional.
     """
+    if fdg_file is None:
+        return top()
+    fdg_file = _get_fdg_file(fdg_file)
+    if not fdg_file:
+        return
```

Comment:
We should log rather than silently returning from here

Reply:
done